<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgSettingsCheats</class>
 <widget class="QWidget" name="wdgSettingsCheats">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>54</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Cheats Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_wdgSettingsCheats">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout_Cheats">
     <item row="2" column="0" colspan="2">
      <widget class="wdgCheatsEditor" name="widget_Cheats_Editor" native="true">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_Cheats_settings">
       <item>
        <widget class="QLabel" name="icon_Cheats_settings">
         <property name="text">
          <string notr="true">Settings icon</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Cheats_settings">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Settings</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_Cheats_settings">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QWidget" name="widget_Cheats_Mode" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_cm">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_Cheats_Mode_disabled">
          <property name="text">
           <string>Disabled</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Cheats_Mode_gg">
          <property name="text">
           <string>Game Genie Rom Mode</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Cheats_Mode_list">
          <property name="text">
           <string>Cheats List Mode</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wdgCheatsEditor</class>
   <extends>QWidget</extends>
   <header>wdgCheatsEditor.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButton_Cheats_Mode_disabled</tabstop>
  <tabstop>pushButton_Cheats_Mode_gg</tabstop>
  <tabstop>pushButton_Cheats_Mode_list</tabstop>
  <tabstop>widget_Cheats_Editor</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
